packages/common/src/discussions/fetch.ts

```diff
@@ -45,6 +46,10 @@ export async function convertItemToDiscussion(
   requestOptions: IRequestOptions
 ): Promise<IHubDiscussion> {
   const model = await fetchModelFromItem(item, requestOptions);
+  await fetchSetting({ id: item.id }).then((settings) => {
```

Comment:
We need to update `requestOptions` arg type to `IHubRequestOptions` in both `fetchDiscussion` and `convertItemToDiscussion` and spread it onto the object passed to `fetchSetting` for auth to work for the settings API request. We should set the setting defaults if the call to fetchSetting fail (no record exists).

```jsx
export async function fetchDiscussion(
  identifier: string,
  requestOptions: IHubRequestOptions
): Promise<IHubDiscussion> {

// ...

export async function convertItemToDiscussion(
  item: IItem,
  requestOptions: IHubRequestOptions
): Promise<IHubDiscussion> {
  const model = await fetchModelFromItem(item, requestOptions);
  let entitySettings;
  try {
    entitySettings = await fetchSetting({ id: item.id, ...requestOptions });
  } catch (e) {
    entitySettings = getDefaultEntitySettings('discussion');
  }
  model.entitySettings = entitySettings;
  const mapper = new PropertyMapper<Partial<IHubDiscussion>, IModel>(
    getPropertyMap()
  );
  const discussion = mapper.storeToEntity(model, {}) as IHubDiscussion;
  return computeProps(model, discussion, requestOptions);
}
```